### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by lusbenjamin**
### Before
image

### After
image

### Notes
- Language upgrades to babel/TS/eslint/prettier now fully support nullish and coalesce!
- Prettier and eslint tool pipeline now properly handles `.jsx`,`.ts`, and `.tsx`
- fixes almost all `npm audit` issues
- Most changes in this PR are due to unimportant `prettier` or `lint` tweaks
- removed `conf` and `acorn` packages
- removed unnecessary plumbing that was redundant with `electron-debug`
- incrementally manually tested things each step of the way
- Left behind `electron` and `interactjs` because upgrading them broke things

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -68,15 +69,8 @@ app.on("ready", () => {
       dsn: "https://4ec87bda1b064120a878eada5fc0b10f@sentry.io/1778171"
     });
     startApp();
-    require("devtron").install();
```
Comment (lusbenjamin) [inline src/main.js]: `electron-debug` does this automatically

File: src/main.js [hunk 2]
```diff
@@ -68,15 +69,8 @@ app.on("ready", () => {
...
     const dotenv = require("dotenv");
     dotenv.config();
-    if (process.env.REACTDEVTOOLSEXT) {
-      // To enable REACT dev tools createa an .env file
-      // and add REACTDEVTOOLSEXT="path"
-      // where path is the path to your chrome extension folder
-      electron.BrowserWindow.addDevToolsExtension(process.env.REACTDEVTOOLSEXT);
-    }
```
Comment (lusbenjamin) [inline src/main.js]: `electron-debug` does this automatically

File: src/shared/stats-fns.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
-import math from "mathjs";
-
-math.config({ precision: 2000 });
+import { create, all } from "mathjs";
+const config = { precision: 2000 };
+const math = create(all, config);
```
Comment (lusbenjamin) [inline src/shared/stats-fns.js]: handles breaking changes for `mathjs`

**Comment by Manuel-777** [comment]
PR aka; "El bumpito"

**Pull request merged**